test/e2e/metering_manual_install_tests.go

```diff
@@ -107,6 +112,105 @@ func testManualMeteringInstall(
 	assert.NoError(t, err, "capturing logs and uninstalling metering should produce no error")
 }
 
+func createNFSProvisioner(ctx *deployframework.DeployerCtx) error {
+	//Waiting for Metering pods
+	ctx.TargetPodsCount = nonHDFSTargetPodCount
+
+	_, err := createTestingNamespace(ctx.Client, "nfs")
```

Comment:
I would recommend moving this as a constant variable somewhere as we use the "nfs" string throughout and this could help improve readability/maintenance by using a constant variable instead.

```go
const nfsTestingNamespace = "nfs"
```